internal/apis/certmanager/types_certificate.go

```diff
@@ -247,6 +254,17 @@ type CertificateSpec struct {
 	NameConstraints *NameConstraints
 }
 
+type OtherName struct {
+	// OID is the object identifier for the otherName SAN.
+	// The object identifier must be expressed as a dotted string, for
+	// example, "1.2.840.113556.1.4.221".
+	OID string `json:"oid,omitempty"`
+
+	// utf8Value is the string value of the otherName SAN.
+	// The utf8Value accepts any valid UTF8 string to set as value for the otherName SAN.
```

Comment:
suggestion: In the `OtherNames` comment we're careful to point out that we don't do validation - I think it'd be worth pointing out here too.

```suggestion
	// utf8Value is the string value of the otherName SAN. Any UTF-8 string can be used, but no
	// validation is performed.
```